### PR TITLE
BooleanRegularExpression Spec

### PR DESCRIPTION
**Pull request by ggetz**
Added spec for `BooleanRegularExpression`

### CODE REVIEW

File: Specs/Scene/BooleanRegularExpressionSpec.js [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+    function MockStyleEngine() {
+    }
+
+    MockStyleEngine.prototype.makeDirty = function() {
+    };
+
+    function Feature() {
+    }
+    Feature.prototype.getProperty = function() {
```
Comment (pjcozzi) [inline Specs/Scene/BooleanRegularExpressionSpec.js]: Arguable, but add vertical whitespace here just like other prototype functions.

File: Specs/Scene/BooleanRegularExpressionSpec.js [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+        BooleanRegularExpression) {
+    'use strict';
+
+    function MockStyleEngine() {
+    }
+
+    MockStyleEngine.prototype.makeDirty = function() {
+    };
+
+    function Feature() {
```
Comment (pjcozzi) [inline Specs/Scene/BooleanRegularExpressionSpec.js]: Probably name this `MockFeature`.

**Comment by pjcozzi** [comment]
Just those trivial comments.  Very nice work here!

**Comment by ggetz** [comment]
Updated!

**Comment by pjcozzi** [comment]
Nice!

**Pull request merged**